juego.py

```diff
@@ -275,7 +273,6 @@ def main(self):
 
     def choose_level(self):
         sonido_menu = load_sound("menu.ogg")
-        menu_sound_played = False
         facil = Juego_button("easy", self.fuente_130, 470, 180)
         medio = Juego_button("medium", self.fuente_130, 417, 360)
         dificil = Juego_button("hard", self.fuente_130, 465, 540)
```

Comment:
94096da introduced `menu_sound_played` and it has its use so removing it is counter intuitive.

Reply:
Yes, that's odd.  It isn't a global.  @marsian83, what is this for?  It causes `F841 local variable 'menu_sound_played' is assigned to but never used`

Reply:
PR #35 was supposed to be a PR fixing the bug where hovering over the buttons made the `sonido_menu` sound play indefinitely. This variable was added to fix this issue. now this is just a residue of the method I initially tried to fix the problem. The PR later turned into a complete refactor of the buttons and thus this variable was supposed to be removed. 
It was left in by accident and I reckon should be removed

Reply:
Thanks!